src/custom-elements/src/components/search-input/search-input.tsx

```diff
@@ -84,7 +84,11 @@ export class SearchInput {
   /**
    * Triggered when the user clicked the Search button.
    */
-  @Event({ eventName: 'chiSearch' }) eventSearch: EventEmitter;
+  @Event({ eventName: 'chiSearch' }) eventSearch: EventEmitter<string>;
+  /**
+   * Triggered when the user clicked in one of the autocomplete elements
+   */
+  @Event({ eventName: `chiSearchInputItemSelected`}) itemSelected: EventEmitter<DropdownMenuItem>
```

Comment:
Single quotes. Is this name correct: `chiSearchInputItemSelected`? @jllr 

```suggestion
  @Event({ eventName: 'chiSearchInputItemSelected'}) eventItemSelected: EventEmitter<DropdownMenuItem>
```